src/commands/guildLog.ts

```diff
@@ -0,0 +1,189 @@
+import { ApplicationCommandOptionType, EmbedBuilder } from "discord.js";
+import { Command } from "../interfaces/Command";
+import { BotEvents } from "mineflayer";
+
+export default {
+	data: {
+		name: "guildlog",
+		description: "Get the guild log!",
+		options: [
+			{
+				name: "member",
+				description: "What user do you want to check? - Optional",
+				type: ApplicationCommandOptionType.String,
+			},
+			{
+				name: "page",
+				description: "What page do you want to check? - Optional",
+				type: ApplicationCommandOptionType.String,
+			},
+		],
+	},
+	run: async (bot, interaction, args) => {
+		const member: string = args[0];
+		const page: number = args[1];
+
+		if ((member && page) == undefined) {
+			try {
+				await bot.executeTask(`/g log`);
+				const gLogArray: string[] = [];
+				let chatListener: BotEvents["message"];
+				const regex =
```

Comment:
It might be worth splitting this into a few smaller regex chunks so it's easier to debug and manage

Reply:
No, because I want it to test on the entire regex at once.